### ISSUE TITLE
[Add Patient] Incorrect email format error may imply that it is an required field

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5829]
## Area or Component

Patients > [Add individual patient](https://demo.simplereport.gov/app/add-patient?facility=a39ef938-d4af-4632-b0c4-7b03af3dc0f5)
## Quick Summary

While adding a patient, if the format of email is incorrect the error says that the field is "Missing or incorrectly formatted" even though the field is not required. 

## Accessibility Violation

The error is not clear. 

## Action Requested

Change the error to say that the format of the email is incorrect.

## Screenshots (optional)
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/8168addf-ee27-438a-96f4-37da6160591d)

**Comment by mehansen** [comment]
Check in with @lauraykerr on updated error message
